### DISCUSSION TITLE
Designate a resource as External (3.5.1 HTML5)

### DISCUSSION THREAD

**Discussion by IPcorps** [godotengine/godot-proposals#5401] [Editor]
For example, I have a background in the scene, to which I assign a texture in the editor. But. I don't want this texture to be exported (included in pck) at the same time because I'm going to load it dynamically. It would be nice to be able to mark a resource in the editor as External (boolean flag) so that it can be seen during development, and at the same time it is not included in the export and does not throw an error.

So far, I see only two ways. Either completely disable error output after export, or delete all "dynamic" resources in the editor before export. Both options not so hot.

**Comment by IPcorps** [comment]
I solved the issue of writing an addon that removes the resources specified for it when exporting. But it would still not be bad if it were possible to specify the resources that should be present in the scene when developing the game in the editor, but should not be in the pck file after export, since they will be dynamically loaded by the game.

**Comment by theraot** [comment]
The Resources tab of the Export preset gives you control over what resources are exported.
